napari/utils/_tests/test_key_bindings.py

```diff
@@ -337,15 +298,18 @@ def add_then_subtract(x):
 
     class Baz(KeymapProvider):
         aliiiens = 0
-        class_keymap = {'A': make_42, 'Control-Shift-B': add_then_subtract}
+        class_keymap = {
+            KeyCode.Shift: make_42,
```

Comment:
curious why is this test case changed from A to shift?

Reply:
decided I wanted to check how it handles "special" keys as there are already multiple test cases for simple letters